lib/SIL/Parser/ParseSIL.cpp

```diff
@@ -2944,7 +2945,7 @@ bool SILParser::parseSpecificSILInstruction(SILBuilder &B,
 
     ValueOwnershipKind forwardingOwnership = Val.getOwnershipKind();
     if (parseForwardingOwnershipKind(forwardingOwnership) ||
-        parseSILDebugLocation(InstLoc, B))
+        parseSILDebugLocation(InstLoc, B, true))
```

Comment:
In case anyone else also wonders, the `true` here is for parseComma.
In other places we often write this as `parseSILDebugLocation(InstLoc, B, /*parseComma=*/ true))` to make it easier to read.